### ISSUE TITLE
I improving the recognition rate about TensorFlow 2 programmer,Google

### ISSUE THREAD

**Issue by nongxi** [tensorflow/tensorflow#30788]
<em>Please make sure that this is a build/installation issue. As per our [GitHub Policy](https://github.com/tensorflow/tensorflow/blob/master/ISSUES.md), we only address code/doc bugs, performance issues, feature requests and build/installation issues on GitHub. tag:build_template</em>

**System information**
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):Ubuntu 18.04 x64
- TensorFlow installed from (source or binary):binary
- TensorFlow version:Tensorflow 2.0.0-beta0
- Python version:Python 3.6
- Installed using virtualenv? pip? conda?:Activate virtualenv,then pip install TensorFlow
- CUDA/cuDNN version:
- GPU model and memory: 1 vCore,1024RAM,Bandwidth 1000G 

**Describe the problem**
I improving the recognition rate, I mean My train the model have hight sussessful than the guide
**Provide the exact sequence of commands / steps that you executed before running into the problem**
URL:https://github.com/nongxi/Machine-Learning-tensorfolw-/blob/master/TensorFlow%202.py
code:
import tensorflow as tf
mnist = tf.keras.datasets.mnist
(x_train, y_train), (x_test, y_test) = mnist.load_data()
x_train, x_test = x_train / 255.0, x_test / 255.0

model = tf.keras.models.Sequential([
  tf.keras.layers.Flatten(input_shape=(28, 28)),
  tf.keras.layers.Dense(128, activation='sigmoid),
  tf.keras.layers.Dropout(0.2),
  tf.keras.layers.Dense(10, activation='softmax')
])

model.compile(optimizer='adam',
              loss='sparse_categorical_crossentropy',
              metrics=['accuracy'])

model.fit(x_train, y_train, epochs=5)
model.evaluate(x_test, y_test)

**Any other info / logs**
Include any logs or source code that would be helpful to diagnose the problem. If including tracebacks, please include the full traceback. Large logs and files should be attached.
log:
Train on 60000 samples
Epoch 1/5
2019-07-17 02:51:34.258226: E tensorflow/core/grappler/optimizers/meta_optimizer.cc:502] constant folding failed: Invalid argument: Unsupported type: 21
2019-07-17 02:51:34.287935: E tensorflow/core/grappler/optimizers/meta_optimizer.cc:502] constant folding failed: Invalid argument: Unsupported type: 21
60000/60000 [==============================] - 4s 66us/sample - loss: 0.0537 - accuracy: 0.9822
Epoch 2/5
60000/60000 [==============================] - 4s 73us/sample - loss: 0.0475 - accuracy: 0.9845
Epoch 3/5
60000/60000 [==============================] - 5s 77us/sample - loss: 0.0451 - accuracy: 0.9852
Epoch 4/5
60000/60000 [==============================] - 4s 72us/sample - loss: 0.0421 - accuracy: 0.9858
Epoch 5/5
60000/60000 [==============================] - 4s 71us/sample - loss: 0.0412 - accuracy: 0.9860
<tensorflow.python.keras.callbacks.History object at 0x7fec4cdd90b8>
>>> model.evaluate(x_test, y_test)
10000/10000 [==============================] - 1s 51us/sample - loss: 0.0678 - accuracy: 0.9813
[0.06783692461897153, 0.9813]

**Comment by pandrey-fr** [comment]
Hi,

No offence, but your post makes no sense. Your reported results suggest that either you had an extremely lucky choice of initialization weights (which could be the case, and is nothing of real interest, especially as it cannot be easily reproduced since you did not set a seed), or you basically trained the model multiple times in a row (i.e. increased the number of epochs) - this is typically the loss and accuracy reached around 10th to 15th epoch on this dataset with this architecture.

In any case, this is not an "issue". Apart from the fact that I highly doubt those are results on the first few epochs, there is a natural variability to the metrics reached (given that there is some randomness in initial weights' generation and dropout processes), so I do not see what it is you are trying to express with this post.

By the way, there is a typo in your code. (missing `'` line 8)

**Comment by ravikyram** [comment]
@nongxi 
Glad to see the improved results.I am closing this issue as this is not a bug or feature request.Thanks!
